e2e/actions/l2_challenger_test.go

```diff
@@ -133,9 +152,14 @@ func TestChallenger(gt *testing.T) {
 	require.NotEqual(t, eth.Bytes32(outputOnL1.OutputRoot), outputComputed.OutputRoot, "output roots must different")
 
 	// deposit bond for challenger
-	challenger.ActDeposit(t, 1_000)
+	challenger.ActDeposit(t, challengerInitialAmount)
 	includeL1Block(t, miner, challenger.address)
 
+	// check bond amount before create challenge
+	bond, err := valPoolContract.GetBond(nil, outputIndex)
+	require.NoError(t, err)
+	require.Equal(t, 0, big.NewInt(dp.DeployConfig.ValidatorPoolMinBondAmount.ToInt().Int64()).Cmp(bond.Amount))
```

Comment:
(Same in `TestChallengerChallengerTimeout` function.)
```suggestion
	require.Equal(t, dp.DeployConfig.ValidatorPoolMinBondAmount.ToInt(), bond.Amount)
```